tests/Unit/RpcLoggerTest.php

```
@@ -354,10 +357,357 @@ public function testLogWithComplexContext(): void
+            {
+                return 'first:' . $value;
+            }
+        };
+
+        $secondProcessor = new class implements ObjectProcessor {
+            public function canProcess(mixed $value): bool
+            {
+                return \is_int($value);
+            }
+
+            public function process(mixed $value, callable $processor): mixed
```

PHPMD (2.15.0)

Avoid unused parameters such as '$processor'. (undefined)

(UnusedFormalParameter)